### DISCUSSION TITLE
KLayers aws layer version 18 doesn't exist

### DISCUSSION THREAD

**Discussion by mishraRamashankar** [explosion/spaCy#12666] [Help: Other Questions] [answered]
While attaching the Spacy layer by KLayers, I'm getting the below error.
**Failed to load layer version details: The resource you requested does not exist.**

I'm on Mumbai region of AWS.
Kindly check and let me know, what is causing this issue.

**Comment by vinbo8** [comment, marked as answer]
This looks like it has more to do with the [KLayers](https://github.com/keithrozario/Klayers) package, so I'd ask about it there.
